### ISSUE TITLE
Precaching cifti atlas in docker

### ISSUE THREAD

**Issue by ZhifangYe** [poldracklab/fmriprep#1145]
When output CIFTI files, I found it needs to download atlas files from OSF server. Many HPC computational nodes like ours doesn't have active Internet connection. I think it's reasonable to prepare those files in the container instead of downloading during running.

**Comment by effigies** [comment]
Agreed. Would you like to submit a PR?

**Comment by ZhifangYe** [comment]
Yeah, I could make some time for a PR this weekend.

**Comment by chrisfilo** [comment]
Great - we much appreciate your contribution!

**Comment by ZhifangYe** [comment]
@effigies I realize that the oasis_dtk31 atlas folder on OSF storage is named `oasis_trt20_DKT31-MNI152NLin` instead of the key used in niworkflows (`oasis_dkt31_mni152`). This leads to `niworkflows.data.get_oasis_dkt31_mni152` shows undesired behaviors and hard to parse the atlas filename.

I think the simplest solution here is rename the folder inside `oasis_trt20_DKT31-MNI152NLin.tar.gz `. I'm not sure whether I could modify file on OSF storage. Could you help me re-upload it?

**Comment by effigies** [comment]
I'm a little surprised, as this seems like something that should have been caught before, as we do have some people using the pipeline. @mgxd perhaps you can comment, here?

@ZhifangYe It might help if you describe the approach you're taking. Just to be clear, what I was suggesting was to add a few lines to `docker/scripts/get_templates.sh`.

**Comment by ZhifangYe** [comment]
Indeed I tried to add the atlas file address in `docker/scripts/get_templates.sh` at first.  But it can't prevent the attempt to re-download files from OSF.

When I pre-cached the atlas, the extracted folder name is `oasis_trt20_DKT31-MNI152NLin`. The code use this atlas is in line 134. Because the folder name is hard coded in line 135, it will not cause any problem during runtime. But it still tries to connect OSF server.
https://github.com/poldracklab/fmriprep/blob/5dd78197a828de566d754b327f0f7d663bfe42f0/fmriprep/interfaces/cifti.py#L133-L138

If I'm right, the expected atlas data should be in path `$CRN_SHARED_DATA/oasis_dkt31_mni152` instead of current `oasis_trt20_DKT31-MNI152NLin`. During running, line 134 will always try to download the tar file even the files are already in `oasis_trt20_DKT31-MNI152NLin`. 

I propose to change line 134  to `label_template = get_oasis_dkt31_mni152()` from `niworkflows.data` and change the folder name inside the tarball in order to respect naming convention in `niworkflows.data`. This change could prevent any attempts to connect Internet.

**Comment by effigies** [comment]
Hi @ZhifangYe, sorry to be so slow about getting back to you. I think your proposal sounds perfectly reasonable.

@mgxd @oesteban Do you see any potential problems? I know @oesteban is planning to update the naming conventions around templates, but I'm not sure what the status on that is, and how much that will interfere.

**Comment by mgxd** [comment]
hi @ZhifangYe @effigies sorry for the radio silence - you are right that the atlas name is not consistent with the osf/niworkflows name, I will submit a fix for this. As for preparing the files in the container, I'm fine with that - perhaps adding it to `/niworkflows_data`

**Comment by effigies** [comment]
@mgxd Great. Yes, please do add it to `/niworkflows_data` using the `docker/scripts/get_templates.sh` script.

**Comment by mgxd** [comment]
@ZhifangYe 

I pushed a new folder `tpl-OASISTRT20` to the OSF storage, which includes the label file `tpl-OASISTRT20_variant-DKT31_space-MNI152NLin2009cAsym.nii.gz`

**Comment by ZhifangYe** [comment]
@mgxd Thanks.

@effigies I submit a PR using newly uploaded template file. And another PR to niworkflows in order to respect the new naming convention.

**Comment by effigies** [comment]
Should be fixed in 1.1.1.
